extensions/3DTILES_implicit_tiling_scheme/README.md

```diff
@@ -0,0 +1,588 @@
+querying tree structure from the server, data storage on the client, as well as simplification and optimization of algorithms involved with the structure of the tree such as traversal,
+visibility, arbitrary  selection of tiles in a region, ray casting, analysis, etc.
+The tileset.json only needs to supply the information needed so that any tile can derive attributes like `geometricError`, `boundingVolume`, `refine`, etc.
+This cuts down the size and verbosity of the tileset.json. Because the access to information describing any part of tree structure is predictable,
+random access queries on the tree are possible.
+
+Because subdivision is predicable, the only information regarding the tree structure that needs to be queried from the server is whether a tile exists or not.
+Only 1 bit per tile is needed for this. For explicit tilesets, per-tile information is a little over 300 bytes. This simplified approach reduces IO and allows easier interoperability with other standards.
+
+TODO: include a representative image here
+
+## Concepts
+### Implicit Tilling
+Implicit tiling is a term used to convey that the rule for how a tile subdivides into its children is the same for every tile in a tileset.
+The two supported methods for how a tile subdivides are quadree subdivision and octree subdivision.
+
+### Quadtree
+When a tile subdivides in a quadtree, it produces 4 equally sized tiles that fit in the footprint of the original tile, the parent tile.
+The tile is split along two axes picking the midpoint of the bounds along those axes. These axes are the x and y axes.
+
+![](figures/quadtree.png)
+
+### Octree
+When a tile subdivides in an octree, it produces 8 equally sized tiles that fit in the footprint of the original tile, the parent tile.
+The tile is split along all three axes picking the midpoint of the bounds along each axis.
+
+![](figures/octree.png)
+
+### Split Axes
+The property used to specify this subdivision is `splitAxes`.
+It is a number indicating the number of axes split, 2 being a quadtree and 3 being an octree.
+
+### Availability Subtree
+The only information that is needed per tile is whether the tile is available at some location in the tree.
+The full tree of information that expresses all tiles' availability is broken up into subtrees(small portions of the full tree). Since a single bit is needed to hold a tile's availability,
+the subtree of availability is expressed as an array of bytes where each bit holds a tile's availability, i.e. a 1 or 0 indicating that the tile is available or not available, respectively.
+
+### Complete Subtree Levels
+In the `tileset.json`, the `completeSubtreeLevels` property is the number of levels in every subtree.
+Every subtree starts from a single tile, its root, and goes down the number levels indicated by `completeSubtreeLevels`. For example,
+```json
+`completeSubtreeLevels: 10`
+```
+means that if a subtree started at level 0, it would go down to level 9, covering 10 levels.
```

Comment:
It should be more clearly stated that subtrees are aligned to `completeSubtreeLevels`. If `completeSubtreeLevels` is 9 `firstSubtreesWithContent` must start at either level 0, 8, 16, etc. Also provide some context for this choice - that it makes it easier to fuse tilesets or do partial updates.